module/migration.mjs

```diff
@@ -496,7 +498,9 @@ export const migrateEffectData = function(effect, migrationData, { parent }={})
   const updateData = {};
   _migrateDocumentIcon(effect, updateData, {...migrationData, field: "icon"});
   _migrateEffectArmorClass(effect, updateData);
-  _migrateTransferEffect(effect, parent, updateData);
+  if ( foundry.utils.isNewerVersion("3.1.0", effect._stats?.systemVersion) ) {
```

Comment:
It's unfortunate, but I don't believe ActiveEffects have `_stats` in v11. We have the `parent` here, so maybe that's good enough?